dev/io.openliberty.microprofile.telemetry.1.0.internal/bnd.bnd

```diff
@@ -43,15 +43,16 @@ Include-Resource: \
 
 Service-Component: \
     io.openliberty.microprofile.telemetry.internal.ResourceProvider; \
-        implementation:=com.ibm.wsspi.classloading.ResourceProvider; \
-        provide:=com.ibm.wsspi.classloading.ResourceProvider; \
-        configuration-policy:=optional; \
+    implementation:=com.ibm.wsspi.classloading.ResourceProvider; \
+    provide:=com.ibm.wsspi.classloading.ResourceProvider; \
+    configuration-policy:=optional; \
 	properties:="resources=${app-resources}" 
 
 # Need an explicit import for api.common because it gets used by weld proxies
 # created for other API classes
 Import-Package: \
 	io.opentelemetry.api.common,\
+	io.openliberty.microprofile.telemetry.common.*,\
 	*
```

Comment:
I don't think we need an explicit import for `io.openliberty.microprofile.telemetry.common.*` because it should be covered by the `*` import.